lightningd/onchain_control.c

```diff
@@ -518,6 +518,44 @@ static u32 infinite_block_deadline(const struct chain_topology *topo)
 	return get_block_height(topo) + 300;
 }
 
+/**
+ * slow_sweep_dealine -- A smart(-ish) way of estimating sweep fees.
+ *
+ * We use the `close_blockheight` as the basis for the target. We then
+ * target to have the funds back at most 2 weeks after the close,
```

Comment:
This final commit does not belong in this PR?

Reply:
The rebase removed it, because the onchaind PR already made it into `master`.